brambling/models.py

```diff
@@ -1098,6 +1098,10 @@ def get_eventhousing(self):
     def has_dwolla_payments(self):
         return self.transactions.filter(method=Transaction.DWOLLA).exists()
 
+    def awaiting_check_confirmation(self):
```

Comment:
This method concerns me, because it hides what's actually happening (running a query) and makes it easy to call this method without noticing the cost. So for example, this method currently gets called once or twice for each event on a user's dashboard. I'd rather see this as a single query to get up to one unconfirmed check per event in the list, then pass that to the context. Easiest might be to give the context a list of tuples `(event, order, has_unconfirmed_check_payments)` rather than a queryset of events... not sure.